app-shells/loksh/loksh-7.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Linux port of OpenBSD's ksh"
+HOMEPAGE="https://github.com/dimkr/loksh"
+SRC_URI="https://github.com/dimkr/loksh/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+
+DEPEND="sys-libs/ncurses:="
```

Comment:
It doesn't look like it's being linked to ncurses. Not that this matters much, but you can drop the `:=` part.